internal/upload/koji/koji.go

```diff
@@ -106,46 +106,53 @@ type CGImportResult struct {
 	BuildID int `xmlrpc:"build_id"`
 }
 
-func New(server string) (*Koji, error) {
-	k := &Koji{}
-	client, err := xmlrpc.NewClient(server, http.DefaultTransport)
+func Login(server, user, password string) (*Koji, error) {
```

Comment:
Nitpick for future work: Should this be called `New()`?

Reply:
Fixed